test/src/examples_complete_test.go

```diff
@@ -67,6 +83,20 @@ func testExamplesCompleteEnabled(t *testing.T) {
 	defer func() { recover() }()
 	assert.Equal(t, `arn:aws:s3:::`+expectedS3BucketName+`/testprefix/*`, getTestResource(policyString),
 		"Templating of var.additional_bucket_policy failed")
+
+	// Validate that Origins are in Origin Groups
+	originGroups := []OriginGroup{}
+	terraform.OutputStruct(t, terraformOptions, "cf_origin_groups", &originGroups)
+	originIds := terraform.OutputList(t, terraformOptions, "cf_origin_ids")
+	originIdsInGroups := []string{}
+	for _, originGroup := range originGroups {
+		for _, member := range originGroup.Member {
+			originIdsInGroups = append(originIdsInGroups, member.OriginId)
+		}
+	}
+	sort.Strings(originIds)
+	sort.Strings(originIdsInGroups)
+	assert.True(t, reflect.DeepEqual(originIds, originIdsInGroups))
```

Comment:
This block is the same as the block in the `TestExamplesCompleteExtraOrigins`. Could you move this to a helper function and call the same helper function in both tests ?